pandas/core/panel.py

```diff
@@ -34,7 +34,7 @@
 _shared_doc_kwargs = dict(
     axes='items, major_axis, minor_axis',
     klass="Panel",
-    axes_single_arg="{0,1,2,'items','major_axis','minor_axis'}")
```

Comment:
I think this shouldn't be changed, because the name is used in ``__init__`` keyword?